### PR TITLE
Use web-time crate

### PR DESCRIPTION
**Pull request by vivekvpandya**
Use web-time crate (https://github.com/daxpedda/web-time) instead of std version. It works for wasm32 targets too. If target support time in std then it's just an alias to that.

### CODE REVIEW

**Comment by dlubarov** [comment]
Did you try with the `timing` feature off, or is that not a good option as you want the timing breakdown?

File: plonky2/Cargo.toml [hunk 1]
```diff
@@ -34,6 +34,7 @@ serde = { version = "1.0", default-features = false, features = ["derive", "rc"]
 serde_json = "1.0"
 static_assertions = { version = "1.1.0", default-features = false }
 unroll = { version = "0.1.5", default-features = false }
+instant = "0.1"
```
Comment (dlubarov) [inline plonky2/Cargo.toml:37]: I wonder if this breaks `no_std` builds? If so, could make it an optional dependency, with `timing` depending on `dep:instant`

**Comment by vivekvpandya** [comment]
> Did you try with the `timing` feature off, or is that not a good option as you want the timing breakdown?

I tried with `default-features=false, features=["std"]` but it did not worked.

but in any case I think this PR is harmless.

**Review by Nashtare — COMMENTED** [review]
> I tried with default-features=false, features=["std"] but it did not worked.

Yeah switching `std` feature on would break with most WASM targets, as you can't make OS assumptions.

OTOH, I'm not super in favor of bringing into the dependencies a crate that's clearly abandoned (last version was 3y ago, issues and PR opens about problems encountered and not addressed).

**Comment by Nashtare** [comment]
If we're to add full timing support for wasm targets, then I'd suggest using [web-time](https://crates.io/crates/web-time), as this issue properly suggests: https://github.com/sebcrozet/instant/issues/52.

**Comment by vivekvpandya** [comment]
@Nashtare anything more needed to be done for this PR? if not could you please merge it.

**Review by Nashtare — APPROVED** [review]

**Pull request merged**